src/mini_apps/quantum_simulation/circuit_cutting/cc_local_cpu.py

```diff
@@ -12,6 +18,34 @@
 # import pdb
 
 
+timestamp = datetime.datetime.now().strftime("%Y%m%d_%H%M%S")
+
+def kill_processes_by_keyword(keyword):
+    """
+    Kills all processes whose command line contains the specified keyword.
+    """
+    for proc in psutil.process_iter(attrs=["pid", "name", "cmdline"]):
+        try:
+            # Check if the process's command line contains the keyword
+            if proc.info["cmdline"] and any(keyword in arg for arg in proc.info["cmdline"]):
+                pid = proc.info["pid"]
+                print(f"Killing process {pid} ({proc.info['name']}) with command: {proc.info['cmdline']}")
+                proc.kill()
+        except (psutil.NoSuchProcess, psutil.AccessDenied):
+            # Process may have terminated or we don't have permissions
+            pass
+
+def stop_ray():
```

Comment:
Pilot-Quantum when starts do ray - stop.. 


https://github.com/radical-cybertools/pilot-quantum/blob/ray-v2/pilot/plugins/ray_v2/cluster.py#L36

Reply:
i am not sure why we need this here again?